src/twisted/internet/test/test_endpoints.py

```diff
@@ -3259,6 +3259,27 @@ def test_sslNoTrailingNewlinePem(self):
         ctx = server._sslContextFactory.getContext()
         self.assertIsInstance(ctx, ContextType)
 
+    @skipIf(skipSSL, skipSSLReason)
+    def test_sslWithCustomCipher(self):
+        """
+        A cipher list is supported, using the OpenSSL format.
+        The colon (:) from the OpenSSL format is replaced with a comma (,).
+        """
+        reactor = object()
+        server = (
+            endpoints.serverFromString(
+                reactor,
+                "ssl:1234:backlog=12:privateKey=%s:"
+                "certKey=%s:cipher=ECDHE-ECDSA-AES128-GCM-SHA256,ECDHE-RSA-AES128-GCM-SHA256,ALL,!ADH,"
+                "@STRENGTH,+RSA,-DSA,SHA1+DES"
+                % (escapedPEMPathName, escapedPEMPathName),
+            ),
+        )
+        self.assertIsInstance(server, endpoints.SSL4ServerEndpoint)
+        self.assertEqual(server._sslContextFactory.method, TLSv1_2_METHOD)
+        ctx = server._sslContextFactory.getContext()
+        self.assertIsInstance(ctx, ContextType)
```

Comment:
This coverage warning indicates that this test always fails.

Reply:
@glyph thankyou, I had to reinstall and run the test for this to show, it was actually passing all the tests locally and I was unable to figure